### PR TITLE
Code updates for SS 8.4.10

### PR DESCRIPTION
**Pull request by ginbot86**
(old title "Mention 'log' and 'logprint' commands in Changelog; Readme version update")

### CODE REVIEW

**Comment by ginbot86** [comment]
Version is now bumped up to 8.4.10.

**Comment by felixwiemuth** [comment]
Thank you! But to be clear: Also only update the version in all files if this version is to be released. If you just code sth. that should be released with a next version (but other stuff may come, too) you can add changes to the changelog but don´t change the program version.

We can release this version soon if you tested it (I´ll merge the pull request then).

**Comment by iBotPeaches** [comment]
I don't think this change warrants a new version increase. There are 2 things left to do that should be done for another version number is increased. 

EDIT: I mean the changes can be "staged"  to .10, but we shouldn't release already with only 1 change. Unless it was a critical hotfix.

1) The "scoreboard" feature needs to pass thru SS to vanilla. It currently doesn't, thus that feature is useless.
2) Anytime you use a teleporter it says "### joined the game" then it timesout after a few seconds.

We don't need to increase # number after a little feature change.

**Comment by ginbot86** [comment]
Got a point. This one can probably wait until some other fixes to SS get through. I'm currently looking at trying to get the 'execsvrcmd' command in events to be workable with SS commands like setgroup (events are where I'm putting a bunch of my attention on as my server uses it extensively).

Edit: As such, since there are more updates planned before we switch to 8.4.10, if nobody objects to it, I'll close off this pull request for later.

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,5 @@
+== 8.4.10 ==
+ * Added 'log' and 'logprint' commands for events
```
Comment (iBotPeaches) [inline CHANGES:2]: I might of messed this up, but there isn't a newline between this version and the previous one (I think the previous one is also affected). If you can enter those newlines and make this look "pretty" (sorry im OCD). We can merge this in, and start working on some fixes / features for 8.4.10

**Comment by ginbot86** [comment]
I've added a new postfix operator for events that allows an event to get the IP of a player. The next thing on my to-do list is for command forwarding for the execsvrcmd event command.

Also, I've changed the topic name for this issue. As previously said, we could simply put in changes and list them in the changelog but not change the official version number yet.

**Comment by iBotPeaches** [comment]
Will do. I'm gonna fix up the "scoreboard" command, and then do some extensive testing before I tackle some bug reports.

**Comment by ginbot86** [comment]
I'm closing this pull request as we're now at 8.4.10. I'll put the getip postfix operator along with some other work on the event host after I do some more testing on it.

I'm also working on getting Bukkit's messages to play nicely with SimpleServer. SS doesn't seem to pick up on Bukkit's player join/leave messages so bots and such end up showing in chat.

**Pull request closed without merging**